server.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const weather = require('./data/weather.json');
+
+const app = express();
+
+app.use(cors());
+
+const PORT = process.env.PORT;
+
+app.listen(PORT, () => console.log('server is listening on port', PORT));
+
+//routes
+app.get('/weather', handleGetWeather);
+
+class Forcast {
```

Comment:
Really great work on the lab!

- There is a typo here, (should read `Forecast`). Just a heads-up to spell it the same way (so far you have so we should be okay, this just caused some bugs in the past for folks) 👍